docs/ext/commands/cogs.rst

```diff
@@ -5,15 +5,8 @@
 Cogs
 ====
 
-There comes a point in your bot's development when you want to organize a collection of commands, listeners, and some state into one class. Cogs allow you to do just that.
-
-The gist:
-
-- Each cog is a Python class that subclasses :class:`.commands.Cog`.
-- Every command is marked with the :func:`.commands.command` decorator.
```

Comment:
> - Every command is marked with the :func:`.commands.command` decorator.

This should probably be documented somewhere, it seems like it's not.

Also, maybe regular Cog docs should link to commands.Cog telling people to look there for ext commands. Because of the naming, I could see it confusing some people.